u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AndroidKeyStoreAttestation.java

```diff
@@ -135,11 +140,15 @@ private AndroidKeyStoreAttestation(Integer keymasterVersion, byte[] attestationC
    *       patchMonthYear  INTEGER,
    *   }
    */
-  public static AndroidKeyStoreAttestation Parse(X509Certificate cert)
-      throws CertificateParsingException {
-    // Extract the extension from the certificate
+  public static AndroidKeyStoreAttestation Parse(
+      X509Certificate[] certChain, X509Certificate[] caCerts) throws CertificateParsingException {
```

Comment:
Yeah... I completely agree.  I added a bit of commentary about one of the checks.  I agree we need to figure out how to make this more clear.